src/Symfony/Component/Routing/Loader/AnnotationFileLoader.php

```diff
@@ -92,6 +92,11 @@ protected function findClass($file)
         $class = false;
         $namespace = false;
         $tokens = token_get_all(file_get_contents($file));
+
+        if (1 === count($tokens) && T_INLINE_HTML === $tokens[0][0]) {
```

Comment:
In this case, as it's the first item, only two options are available: inline HTML or PHP start tag. These both are arrays, so we don't have to worry about it.

Btw, I just discovered that we can simplify this by only doing `count($tokens) === 1`, just `<?php` returns T_INLINE_HTML with `<?php` as value.